src/Component/FileImportButton.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import Button from "@material-ui/core/Button";
+
+type FileImportButtonProps = {
+  id: string;
+  buttonText: string;
+  fileType: string;
+  onChange?: (e: React.ChangeEvent<HTMLInputElement>) => void;
+};
+
+export const FileImportButton = ({
+  id,
+  buttonText,
+  fileType,
+  onChange
+}: FileImportButtonProps) => (
+  <React.Fragment>

```

Comment:
I like using `<>` instead of `<React.Fragment>`, but they are functionally identical. Total style preference though, use whichever you like.

Reply:
I started preferring `<> ` but have since moved more towards keeping the `React` namespace for things like Fragment, useState, useEffect, etc. to make super explicit the source of those imports (and also not require the additional `import React, { useState, useEffect } from 'react'` detail). Definitely 100% style preference though.

Reply:
Keeping <React.Fragment>